### PR TITLE
Fix inferentia integration tests

### PR DESCRIPTION
**Pull request by michaelbeaumont**
### Description
Broken since https://github.com/weaveworks/eksctl/pull/3227

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: integration/tests/inferentia/inferentia_test.go [hunk 1]
```diff
@@ -170,18 +172,18 @@ var _ = Describe("(Integration) Inferentia nodes", func() {
...
 
-func newClientSet() *kubernetes.Clientset {
+func newClientSet(name string) *kubernetes.Clientset {
 	cfg := &api.ClusterConfig{
 		Metadata: &api.ClusterMeta{
-			Name:   defaultCluster,
+			Name:   name,
 			Region: params.Region,
 		},
 	}
```
Comment (aclevername) [inline integration/tests/inferentia/inferentia_test.go:189]: how does this differ to whats currently being done? Both `params` and `defaultCluster` are variables defined in the suite, and populated in the `init`. I feel like I'm missing something

Comment (michaelbeaumont) [reply integration/tests/inferentia/inferentia_test.go:189]: there are more than one cluster created:
https://github.com/weaveworks/eksctl/pull/3259/files#diff-089af4f1a66bd9a4c22a730918fe1810f044e1b910ec017506773e58f9ac59d9R117

Comment (aclevername) [reply integration/tests/inferentia/inferentia_test.go:189]: ahhhh 👍

**Review by aclevername — APPROVED** [review]
LGTM!

**Pull request merged**